examples/niscope/graph-measurement-betterproto.py

```diff
@@ -189,4 +189,6 @@ async def main():
     await CloseGrpcScope(*grpc_scope)
 
 ## Run the main
-asyncio.run(main())
+loop = asyncio.get_event_loop()
```

Comment:
Use this method of starting an async loop so that it doesn't throw when we exit inside the await